routers/web/repo/issue.go

```diff
@@ -3581,6 +3581,11 @@ func IssuePosters(ctx *context.Context) {
 	issuePosters(ctx, false)
 }
 
+// IssuePostersWithoutRepo clone of IssuePosters but without repo
+func IssuePostersWithoutRepo(ctx *context.Context) {
+	issuePostersWithoutRepo(ctx)
```

Comment:
Why we need another function?

Reply:
IssuePoster -> issuePosters -> GetIssuePostersWithSearch 

`->` is calls

GetIssuePostersWithSearch has  `repo *Repository` as argument, but we dont have repo info in `/issues`

thats why i added almost duplicate func for each 

IssuePostersWithoutRepo -> issuePostersWithoutRepo -> GetIssuePostersWithSearchWithoutRepo

all these funcs are duplicate of their 'with repo' counterpart but without any repo info